moped-editor/src/views/projects/projectView/ProjectComponents/utils/makeFeatures.js

```diff
@@ -0,0 +1,188 @@
+import { getDrawId } from "./features";
+
+/*
+ * When we insert in the DB we need to translate the property names that either come from a CTN layer
+ * or the draw tools into columns names in the database.
+ * This is also used to track all layers of data present in the map.
+ */
+export const featureTableFieldMap = {
```

Comment:
Any commented out column names here with `NULL` as the value are columns that I think that we can remove from the DB tables. I noticed that we don't have any values populated there in the transpose script output.

@johnclary @frankhereford One question that I have here is about `symbol` for the `feature_street_segments` table. I saw `0` and `2` in that columns, but I'm not sure where that is coming from or what it represents.

Reply:
@mddilley wrote: 
> One question that I have here is about symbol for the feature_street_segments table. I saw 0 and 2 in that columns, but I'm not sure where that is coming from or what it represents.

👋 Good question. TBH, I suspect that @johnclary or @chiaberry will have a better idea than I do. The various `feature_[name]` tables have the common field(s) inherited from `features` and then the columns that are unique to the feature table -- those were created as needed to handle whatever key-value pairs were in the geoJSON objects. I'm not sure where that data was originally sourced, I just took it at face value and did what I had to do to get it in the DB.

Long way to say - sorry I don't know, but I am hopeful others do. If no one knows, maybe dropping those columns is in order. 🤔

Reply:
yep, it's an artifact of just wholesale copying the geojson properties from whatever layer. i will give those columns a close look and come back with an opinion on what can be dropped.

 i do think we should minimize the columns down to what's essential + what we could reasonably need for a well-defined near-term enhancement, but i can also see us punting that cleanup until a few releases from now. i don't want us to get bogged down patching Frank's script.

Reply:
i just noticed we have the old `project_extent_id` column in `feature_street_segment`. that column theoretically holds the same data as `ctn_segment_id`—we renamed `project_extent_id` to  `ctn_segment_id` at some point and never cleaned up after ourselves :/ 

i will make a backlog issue to handle that in the transpose script and drop `project_extent_id` - does that sound like the right way to deal with it?

Reply:
That sounds perfect to me! `ctn_segment_id` is great since it is easier see that it connects to the source.